resources/config_unified_alerting.rb

```diff
@@ -21,3 +21,9 @@
 use 'partial/_config_file'
 
 property :admin_config_poll_interval_seconds, Integer
```

Comment:
I'm not sure whether this exists at all and whether it was working in general because the path was wrong. See the end of this file. Maybe it can be deleted?